starwars/src/AppStyles.js

```diff
@@ -0,0 +1,63 @@
+import styled from "styled-components";
+
+// font-family: 'Righteous', cursive;
+// font-family: 'Orbitron', sans-serif;
+// font-family: 'Lalezar', cursive;
```

Comment:
What are those? 🕵 Delete commented parts of the code that you don't use anymore. Keep it clean!